typedoc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  "name": "SF-X",
+  "mode": "modules",
+  "out": "./docs",
+  "readme": "none",
```

Comment:
Is there no README we can point to here?

Reply:
If there is no readme in the module (sayt) folder then it points to the parent readme which this flag avoids. @jmbtutor @DanielMoniz-GBI I will make a readme for sayt?

Reply:
The reason we added this in StoreFront (and why it's still relevant here) is that we want the top-level to expose the list of packages in the repo with links to each package's documentation. If the top-level README is exposed, then that list is not displayed, and it's awkward to add them to the README because it doesn't translate well to other contexts. We decided on having this flag in because it's the one that provided the most useful information.

Reply:
The file got deleted from this spot when I moved it to the new config folder. @DanielMoniz-GBI are you ok with this setting?

Reply:
Yes, although I do think we should add a simple comment explaining that we are leaving it as `none` to ensure that a list of READMEs from all packages is displayed.